app/config/config_ci.yml

```diff
@@ -20,6 +20,8 @@ parameters:
       - from_id: old-subject
         to_id: new-subject
         to_name: New Subject
+    rds_articles:
```

Comment:
I think this needs to be in `config_test.yml` too, unless it is stubbed out or that configuration includes this

Reply:
Ah there's no `config_test.yml` (anymore?), let's see what environment PHPUnit uses.

Reply:
By dumping a few things I can confirm this config is always loaded when running `docker-compose run app vendor/bin/phpunit test/Controller/ArticleControllerTest.php`, even locally.

Reply:
Config was working as expected in the end, the problem was in the assertion.